packages/smart-wallet/src/offerWatcher.js

```diff
@@ -220,15 +225,14 @@ export const prepareOfferWatcher = baggage => {
           facets.helper.updateStatus({ payouts: amounts });
         },
         /**
+         * If promise disconnected, watch again. Errors are handled by the resultWatcher.
          * @param {Error | import('@agoric/internal/src/upgrade-api.js').UpgradeDisconnection} reason
          * @param {UserSeat} seat
          */
         onRejected(reason, seat) {
           const { facets } = this;
           if (isUpgradeDisconnection(reason)) {
             void watchForPayout(facets, seat);
-          } else {
-            facets.helper.handleError(reason);
```

Comment:
Please revert this change. I mis-spoke when I said all the calls to `handleError` would get called at the same time. `resultWatcher` gets called separately from `numWantsWatcher and `paymentWatcher`.  So in addition to calling `handleError` from `resultWatcher`, at least one of the others must be called, and this one reads better semantically.